prefect.yaml

```diff
@@ -1,36 +1,22 @@
 name: prefect-collection-registry
-prefect-version: 3.0.0
-
 
 definitions:
   work_pools:
     kubernetes_prd_internal_tools: &kubernetes_prd_internal_tools
       name: kubernetes-prd-internal-tools
       job_variables:
-        image: prefecthq/prefect:3.0.0-python3.12
-        env:
-          EXTRA_PIP_PACKAGES: github3.py fastjsonschema prefect-gcp
-
-
+        image: prefecthq/prefect:3.1.11-python3.12
+        command: uv run --with git+https://github.com/PrefectHQ/prefect-collection-registry.git@main python -m prefect.engine
```

Comment:
Why are you overriding the command here?

Reply:
to install the dependencies for the flow